documentation/ProjectReference-Protocol.md

```diff
@@ -57,7 +57,8 @@ If implementing a project with an “outer” (determine what properties to pass
   * It returns an item with the following metadata:
     * `TargetFrameworks` indicating what TargetFrameworks are available in the project
     * `TargetFrameworkMonikers` and `TargetPlatformMonikers` indicating what framework / platform the `TargetFrameworks` map to.  This is to support implicitly setting the target platform version (for example inferring that `net5.0-windows` means the same as `net5.0-windows7.0`) as well as treating the `TargetFramework` values [as aliases](https://github.com/NuGet/Home/issues/5154)
-    * Boolean metadata for `HasSingleTargetFramework` and `IsRidAgnostic`.
+    * Boolean metadata for `HasSingleTargetFramework` and `IsRidAgnostic` and `IsVcxOrNativeProj`.
+    * `Platforms` indicating what `Platforms` are available for the project to build as.
```

Comment:
Nit: the second use of the word platforms is as an english explanation rather than a recursive definition, so drop the identifier treatment.
```suggestion
    * `Platforms` indicating what platforms are available for the project to build as.
```